sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/_tests_/e2e/student-scholastic-standings.institutions.controller.saveScholasticStanding.e2e-spec.ts

```diff
@@ -341,26 +552,277 @@ describe("StudentScholasticStandingsInstitutionsController(e2e)-saveScholasticSt
     ]);
   });
 
+  it("Should not create new scholastic standing Student did not complete program for a part-time application when date of change is not between study start date and end date.", async () => {
```

Comment:
Instead of having 3 different tests we could iterate through the StudentScholasticStandingChangeType types. 

Something similar to https://github.com/bcgov/SIMS/blob/feature/%232568-ScholasticStanding_Validation/sources/packages/backend/workflow/test/2022-2023/parttime-assessment/disbursements/parttime-assessment-disbursements.e2e-spec.ts#L363

Reply:
For the awards, it is fine to know what award it is failing as the logic is same in all the awards, but using the iteration this case i felt it was not advisable as there are 3 different scholastic standing with different logics and every test case has only one purpose. So created like this. If you want more details we can chat.